test/functional/tools/catDocker.xml

```diff
@@ -1,9 +1,9 @@
 <tool id="catdc" name="Concatenate datasets (in docker)">
     <description>tail-to-head</description>
     <requirements>
-      <container type="docker">busybox:ubuntu-14.04</container>
+      <container type="docker">bgruening/busybox-bash:0.1</container>

```

Comment:
Let's revert the bash related changes in lieu of solving the problem of containers not having bash with https://github.com/galaxyproject/galaxy/pull/2282?